rpc_util.go

```diff
@@ -217,6 +217,18 @@ func Code(err error) codes.Code {
 	return codes.Unknown
 }
 
+// Desc returns the error description of err if it was produced by the rpc system.
+// Otherwise, it returns err.Error() or empty string when err is nil.
+func Desc(err error) string {
```

Comment:
sounds good. Had the same feeling actually. :) changing ...

On Tue, Sep 22, 2015 at 4:49 PM, David Symonds <notifications@github.com>
wrote:

> In rpc_util.go
> <https://github.com/grpc/grpc-go/pull/353#discussion_r40158957>:
>
> > @@ -217,6 +217,18 @@ func Code(err error) codes.Code {
> >  	return codes.Unknown
> >  }
> >
> > +// Desc returns the error description of err if it was produced by the rpc system.
> > +// Otherwise, it returns err.Error() or empty string when err is nil.
> > +func Desc(err error) string {
>
> This isn't a great name in the context. grpc.Desc doesn't say much
> (whereas at least the above grpc.Code is referring to the more specific
> "code" concept).
>
> Perhaps ErrorDesc would be clearer?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc-go/pull/353/files#r40158957>.
>